tools/schema_loader.cc

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright (C) 2021-present ScyllaDB
+ */
+
+/*
+ * This file is part of Scylla.
+ *
+ * Scylla is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Scylla is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <algorithm>
+#include <boost/algorithm/string.hpp>
+#include <seastar/core/fstream.hh>
+#include <seastar/util/closeable.hh>
+
+#include "cql3/query_processor.hh"
+#include "cql3/statements/create_keyspace_statement.hh"
+#include "cql3/statements/create_table_statement.hh"
+#include "cql3/statements/create_type_statement.hh"
+#include "db/config.hh"
+#include "database.hh"
+#include "gms/feature_service.hh"
+#include "locator/token_metadata.hh"
+#include "locator/snitch_base.hh"
+#include "tools/schema_loader.hh"
+#include "utils/fb_utilities.hh"
+
+namespace {
+
+sstring read_file(std::filesystem::path path) {
+    auto file = open_file_dma(path.string(), open_flags::ro).get();
```

Comment:
Use path.native() for OS API.

See https://en.cppreference.com/w/cpp/filesystem/path/native